src/components/signup.jsx

```diff
@@ -1,107 +1,105 @@
-import React, { Component } from "react";
+import React from "react";
+import "bootstrap/dist/css/bootstrap.css";
+import "../index.css";
+import { Link } from "react-router-dom";
 
 const Signup = () => {
   return (
-    <div class="container">
-      <h1 class="row justify-content-center mt-5">ESTATE CITADEL</h1>
-      <div class="row justify-content-center mt-5">
-        <div class="col-md-6">
-          <div class="card">
-            <div class="card-body">
-              <img
-                class="col-md-"
-                src="/images/logo.png"
-                alt="Logo"
-                width="250"
-                height="100"
+    <body>
```

Comment:
Why use the `body` tag here?? Every html page should have a single `body` tag and there's already one in your `index.html` file.

Reply:
Ok Sir